doc/classes/Node.xml

```diff
@@ -1219,5 +1223,14 @@
 		<constant name="INTERNAL_MODE_BACK" value="2" enum="InternalMode">
 			The node will be placed at the end of the parent's children list, after any non-internal sibling.
 		</constant>
+		<constant name="AUTO_TRANSLATE_MODE_INHERIT" value="0" enum="AutoTranslateMode">
+			Inherits [member auto_translate_mode] from the node's parent. This is the default for any newly created node.
+		</constant>
+		<constant name="AUTO_TRANSLATE_MODE_ALWAYS" value="1" enum="AutoTranslateMode">
+			Always automatically translate. This is the inverse of [constant AUTO_TRANSLATE_MODE_DISABLED], and the default for the root node.
+		</constant>
+		<constant name="AUTO_TRANSLATE_MODE_DISABLED" value="2" enum="AutoTranslateMode">
+			Never automatically translate. This is the inverse of [constant AUTO_TRANSLATE_MODE_ALWAYS].
```

Comment:
Should probably mention that makes the node skipped from POT generation (the property already mentions it, but doesn't clarify).

Reply:
Well, I did not know what POT stands for until now, so I think it's not important. The editor always refers to it as POT.
Explaining it in more detail is something for a tutorial documentation page.